res/values-es/strings.xml

```diff
@@ -152,58 +152,58 @@
     <string name="feedback">Retroalimentación</string>
     <string name="favorites">Favoritos</string>
     <string name="favorite_stories_narrow">@string/stories</string>
-    <string name="favorite_stories">Historias Favoritas</string>
-    <string name="favorite_search_narrow">Busquedas</string>
-    <string name="favorite_search">Busquedas Favoritas</string>
+    <string name="favorite_stories">Historias favoritas</string>
+    <string name="favorite_search_narrow">Búsquedas</string>
+    <string name="favorite_search">Busquedas favoritas</string>
     <string name="expires_on">Expira en:</string>
     <string name="empty_recents_title">No hay Recientes:</string>
     <string name="empty_recents_text">Una vez comiences a buscar\no buscar historias, tu actividad\nserá mostrada aquí</string>
-    <string name="empty_favorites_title">No hay Favoritos</string>
+    <string name="empty_favorites_title">Sin favoritos</string>
     <string name="SummaryWaterCooler">Las fuentes crean un flujo de historias interesantes que no vas a encontrar en aplicaciones de noticas regulares. Todas las fuentes son especiales como \”la más compartida\” o \”la más gustada.\” Verifica cuando estés esperando por cinco minutos. Nota: puedes remover Historias desde tu pantalla de inicio utilizando la opción \”Pantalla de Inicio\”</string>
-    <string name="ToastDeleteStoryInHistory">Esta historia a sido removida de tus historial</string>
-    <string name="ToastDeleteUrlInHistory">Este url ha sido removido de tu historial</string>
+    <string name="ToastDeleteStoryInHistory">Esta historia fue removida de tu historial</string>
```

Comment:
this story -> Esta historia fue eliminada de tu historial
the story -> La historia fue eliminada de tu historial